src/common/com/intellij/plugins/haxe/ide/annotator/HaxeSemanticAnnotator.java

```diff
@@ -44,26 +47,35 @@ public void annotate(PsiElement element, AnnotationHolder holder) {
   static void analyzeSingle(final PsiElement element, AnnotationHolder holder) {
     if (element instanceof HaxePackageStatement) {
       PackageChecker.check((HaxePackageStatement)element, holder);
-    } else if (element instanceof HaxeMethod) {
+    }
+    else if (element instanceof HaxeMethod) {
       MethodChecker.check((HaxeMethod)element, holder);
-    } else if (element instanceof HaxeClass) {
+    }
+    else if (element instanceof HaxeClass) {
       ClassChecker.check((HaxeClass)element, holder);
-    } if (element instanceof HaxeType) {
+    }
+    if (element instanceof HaxeType) {
```

Comment:
Didn't notice. Probably I separated it with some empty lines, but when reformatted they were combined as if it was a single if...else chain. I have examined it, and I think they could be all if else. In other case I could always separate it in three methods but I think it is not the case.

I highly agree with you that it should be consistent. I will fix it in next commit.